common/client.go

```diff
@@ -117,6 +117,11 @@ func (c *DatabricksClient) Patch(ctx context.Context, path string, request any)
 	return c.Do(ctx, http.MethodPatch, path, nil, request, nil, c.addApiPrefix)
 }
 
+// Patch on path
+func (c *DatabricksClient) PatchWithResponse(ctx context.Context, path string, request any, response any) error {
```

Comment:
To confirm, FieldMask will be sent as part of `request`, right?

Reply:
Yes, you can see it on the tests.